editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -1052,6 +1056,21 @@ void Polygon2DEditor::_uv_draw() {
 		return;
 	}
 
+	// Force point editing mode if there's no polygon yet.
+	if (node->get_polygon().size() == 0) {
```

Comment:
Why is this code in draw method?

Reply:
I didn't find more proper place for it. I guess I should try to place it in a separate function and figure when to call it. I guess on popup and editing points are all the cases we need, I'll try.